rules-emerging-threats/2023/Exploits/CVE-2023-20198/cisco_syslog_cve_2023_20198_ios_xe_web_ui.yml

```diff
@@ -0,0 +1,27 @@
+title: Exploitation Indicators Of CVE-2023-20198
+id: 2ece8816-b7a0-4d9b-b0e8-ae7ad18bc02b
+status: experimental
+description: Detecting exploitation indicators of CVE-2023-20198 a privilege escalation vulnerability in Cisco IOS XE Software Web UI.
+references:
+    - https://sec.cloudapps.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-iosxe-webui-privesc-j22SaA4z
+author: Lars B. P. Frydenskov (Trifork Security)
+date: 2023/10/20
+tags:
+    - attack.privilege_escalation
+    - attack.initial_access
+logsource:
+    product: cisco
+    service: syslog
```

Comment:
```suggestion
```
It is the syslog message but not the syslog daemon.
%.... is the IOS source log